react/components/TranslatedTitle.tsx

```diff
@@ -3,7 +3,8 @@ import { FormattedMessage } from 'react-intl'
 
 // The b-s token makes the `b` class be applied only on mobile devices.
 const TranslatedTile: FunctionComponent<TranslatedTileProps> = ({ title }) => {
-  const titleText = title || <FormattedMessage id="base.title" />
+  const titleText = title ?? <FormattedMessage id="base.title" />
```

Comment:
If `titleText` can be empty, we should keep the `||` @lucasecdb

Reply:
I think we can keep it with the nullish coalescing